website/src/components/EventSlider/index.tsx

```diff
@@ -155,7 +156,7 @@ const EventSlider: React.FC<EventsProps> = ({
       height="100%"
     >
       <Typography variant="h4" className={classes.noEventText}>
-        {t("community.communityEvents.noEvent.message")}
+      <Link target = "_blank" className = {classes.noEventLink} href = {EXTERNAL_LINKS.CNCF_EVENTS}>{t("community.communityEvents.noEvent.message")}</Link>
```

Comment:
Please remove the extra white spaces, in some other places too (all within links)

Reply:
It is no harm Rakesh. In fact it is a good way to write attributes AFAIK.

Reply:
Done